modules/metering-upgrade-verify.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-upgrading-metering.adoc
+
+[id="metering-upgrade-verify_{context}"]
+= Verifying Metering Upgrade
+
+You can verify the metering upgrade by performing any of the following:
+
+*  Check the Metering Operator ClusterServiceVersion (CSV) for the new metering version through either the web console or by using the `oc` CLI.
+**  Web console:
++
+.Procedure
++
+***  Navigate to *Operators* -> *Installed Operators* in the metering namespace.
+***  Select the Metering Operator to view the *Operator Details* page.
+***  Check the *Installed Version* for the upgraded metering version. The *Starting Version* shows the metering version prior to upgrading.
++
+**  Using the `oc` CLI:
++
+.Procedure
++
+***  Check the Metering Operator CSV:
++
+----
+$ oc get csv
+----
++
+In the following example, the 4.5 Metering Operator upgrade is successful and replaces the 4.4 metering installation:
++
+----
+NAME                                        DISPLAY                  VERSION              REPLACES                               PHASE
+elasticsearch-operator.4.5.0-202006161654   Elasticsearch Operator   4.5.0-202006161654                                          Succeeded
+metering-operator.4.5.0-202006161654        Metering                 4.5.0-202006161654   metering-operator.4.4.0-202005252114   Succeeded
+----
+
+*  Check that all required Pods in the metering namespace are created through either the web console or by using the `oc` CLI.
```

Comment:
If read incorrectly, it might seem like we're telling users here that the Pods must be created through the console or CLI. Technically, that's true, though not what you're suggesting here. :)
So, maybe break into two sentences? 

> Check that all required Pods in the metering namespace are created. This can be done through either the web console or CLI.

(Also, I don't think it's necessary to say "the `oc` CLI` every time.)